content/software/plc-ide/tutorials/can-setup/content.md

```diff
@@ -137,5 +137,5 @@ You can edit the table by adding, removing new variables, you can also assign an
 ### Next Steps
 
 Now that you have completed the setup of the Portenta Machine Control and its CAN bus communication with the Arduino PLC IDE, you can continue with:
-- Plug a CAN sensor to your device and get its data.
+- Plug a CAN sensor into your device and get its data.
```

Comment:
I think saying "to" here is correct, but I could be wrong

Reply:
Recovered